app/frontend/src/components/TabBar/TabBar.tsx

```diff
@@ -23,11 +23,13 @@ export default function TabBar<T extends Record<string, React.ReactNode>>({
   labels,
 }: TabBarProps<T>) {
   const classes = useStyles();
+  const tabContext = useTabContext();
   const handleChange = (event: any, newValue: keyof T) => {
     setValue(newValue);
   };
   return (
     <TabList
+      value={tabContext?.value}
```

Comment:
```suggestion
```
The custom tab select still works for me when I tried this branch locally so I don't think this is necessary.

Reply:
Ah, thanks for pointing that out! Now I understand how this component works :)